specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/extensions/stable/2024-11-01/extensions.json

```diff
@@ -0,0 +1,705 @@
+          "type": "string"
+        }
+      }
+    },
+    "Scope": {
+      "description": "Scope of the extension. It can be either Cluster or Namespace; but not both.",
+      "type": "object",
+      "properties": {
+        "cluster": {
+          "type": "object",
+          "x-nullable": true,
+          "description": "Specifies that the scope of the extension is Cluster",
+          "$ref": "#/definitions/ScopeCluster"
+        },
+        "namespace": {
+          "type": "object",
+          "x-nullable": true,
+          "description": "Specifies that the scope of the extension is Namespace",
+          "$ref": "#/definitions/ScopeNamespace"
+        }
+      }
+    },
+    "ExtensionStatus": {
+      "description": "Status from the extension.",
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "Status code provided by the Extension"
+        },
+        "displayStatus": {
+          "type": "string",
+          "description": "Short description of status of the extension."
+        },
+        "level": {
+          "type": "string",
+          "description": "Level of the status.",
+          "enum": [
+            "Error",
+            "Warning",
+            "Information"
+          ],
+          "default": "Information",
+          "x-ms-enum": {
+            "name": "LevelType",
+            "modelAsString": true
+          }
+        },
+        "message": {
+          "type": "string",
+          "description": "Detailed message of the status from the Extension."
+        },
+        "time": {
+          "type": "string",
+          "description": "DateLiteral (per ISO8601) noting the time of installation status."
+        }
+      }
+    },
+    "Extension": {
+      "description": "The Extension object.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "description": "Properties of an Extension resource",
+          "properties": {
+            "extensionType": {
+              "description": "Type of the Extension, of which this resource is an instance of.  It must be one of the Extension Types registered with Microsoft.KubernetesConfiguration by the Extension publisher.",
+              "type": "string"
+            },
+            "autoUpgradeMinorVersion": {
+              "description": "Flag to note if this extension participates in auto upgrade of minor version, or not.",
+              "type": "boolean",
+              "default": true
+            },
+            "releaseTrain": {
+              "description": "ReleaseTrain this extension participates in for auto-upgrade (e.g. Stable, Preview, etc.) - only if autoUpgradeMinorVersion is 'true'.",
+              "type": "string",
+              "default": "Stable"
```

Comment:
>"default": "Stable" [](http://example.com/codeflow?start=14&length=19)

This seems to be different compared with base.

Reply:
This was one of the validation errors that was flagged, will revert for backward compat with existing service behavior.